.github/workflows/main.yml

```diff
@@ -13,9 +13,16 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0 # avoid shallow clone so nbgv can do its work.
+    - uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 5.0.100
     - uses: nuget/setup-nuget@v1.0.2
     - name: Restore
       run: nuget restore src/BehaviorsSdk.sln
-    - uses: microsoft/setup-msbuild@v1.0.1
-    - name: Build
+    - uses: microsoft/setup-msbuild@v1.0.2
+      with:
+        vs-version: '[16.8,16.9)'
+    - name: Build Solution
```

Comment:
Should this be [16.8,) to indicate anything >= 16.8 is ok?

Reply:
@mgoertz-msft Yes, because of

> The syntax is the same used for Visual Studio extensions, where square brackets like "[" mean inclusive, and parenthesis like "(" mean exclusive. A comma is always required, but eliding the minimum version looks for all older versions and eliding the maximum version looks for all newer versions. See the vswhere wiki for more details.

https://github.com/microsoft/setup-msbuild#specifying-specific-versions-of-visual-studio

But currently the GitHub actions doesn't know v16.8

Reply:
@punker76 Do you mean GitHub actions currently doesn't know anything greater than 16.8 and that's why you currently limit it to 16.8?

Reply:
@mgoertz-msft No, I meant GitHub actions currently only know `16.7.0.37604`

Reply:
GitHub Actions uses now latest VS.